vcr/concept/config.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * Nuts node
+ * Copyright (C) 2021 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package concept
+
+import (
+	"encoding/json"
+	"regexp"
+	"strings"
+
+	"github.com/nuts-foundation/go-did/vc"
+	"github.com/tidwall/gjson"
+)
+
+// Config defines the concept configuration for a VerifiableCredential
+type Config struct {
+	// Concept groups multiple credentials under a single name
+	Concept string `yaml:"concept"`
+	// CredentialType defines the type of the credential. 'VerifiableCredential' is omitted.
+	CredentialType string `yaml:"credentialType"`
+	// Indices contains a set of Index values
+	Indices []Index `yaml:"indices"`
+	// Template is the string template for outputting a credential to a common format
+	// Each <<JSONPath>> value is substituted with the outcome of the JSONPath query
+	Template *string `yaml:"template"`
+}
+
+var templateStringMatcher, _ = regexp.Compile(`<<([a-zA-Z\\.]+)>>`)
```

Comment:
`_` and `-` not allowed? `_` is allowed in JSON field names (think `-` isn't)

Reply:
indeed, `_` should be there.